lib/tachikoma/application.rb

```diff
@@ -99,6 +100,13 @@ def bundler
             @parallel_option
           ].compact))
           sh(*%w(bundle update))
+
+          # restore_bundled_with
+          lock_file_contents = File.read(@bundler_lock_file)
```

Comment:
`sh(['bundle', 'exec', 'restore-bundled-with', '--lockfile', @bundler_lock_file])` does not work I expected. #188